src/igvfd/schemas/analysis_step_version.json

```diff
@@ -69,11 +69,17 @@
         }
     },
     "fuzzy_searchable_fields": [
+        "aliases",
+        "analysis_step.name",
+        "analysis_step.workflow.name",
         "description",
-        "submitted_by.title"
+        "lab.title",
+        "submitted_by.title",
+        "software_versions.name"
```

Comment:
This wasn't mentioned in your Jira comment. Was software_versions.name added intentionally?

Reply:
Yes, I'll add that to my comment.